corehq/messaging/scheduling/forms.py

```diff
@@ -2816,6 +2847,13 @@ class ConditionalAlertScheduleForm(ScheduleForm):
         required=False,
     )
 
+    sms_stale_after = IntegerField(
```

Comment:
nit: can we add a help text that tells user what the default value is? That will just give them an idea of what we consider as stale and have an idea what to think about.
Something like
`default: 7 days` or `default: 168 hours` , the first one is better though, but i know it adds more complexity to logic if we choose to have this value in hours.